configure.sh

```diff
@@ -17,6 +17,8 @@ Configures the CMAKE build environment.
                         Required for interpolant based model checking
 --with-cvc4             build with CVC4 support (default: off)
 --debug                 build debug with debug symbols (default: off)
+--python                compile with python bindings (default: off)
+--py2                   use python2 interpreter (default: python3)
```

Comment:
python2 reached its end-of-life. Should we support it?

Reply:
Hmm that's a good question... I wouldn't be surprised if there are still a couple people using it, but maybe it's not worth the headache. I don't have a specific use case in mind, except that I know Mac can sometimes have issues with python3 bindings (issues linking the right python library).